tests/vcs/git/GitCommitBearTest.py

```diff
@@ -298,16 +298,140 @@ def test_check_issue_reference(self):
 
         # Adding BitBucket remote for testing
         self.run_git_command('remote', 'add', 'test',
-                             'https://bitbucket.com/user/repo.git')
+                             'https://user@bitbucket.org/user/gittest.git')
```

Comment:
Is there a specific reason why the test repo url needs to be changed to this?

Reply:
Oh
I did that to keep the url format consistent with the one suggested by bitbucket on the clone option for a repository

Although the above also redirects fine 

Will change that :sweat_smile: